### PR TITLE
issueno32

### LINKED ISSUE #32: Design view opportunity component for mobile and tab view
image
image
modify the circled part in both images

### PR DESCRIPTION
**Pull request by saumyasrivastava08**
Fixed #32

### CODE REVIEW

**Review by Manvityagi — CHANGES_REQUESTED** [review]
@saumyasrivastava08 
You probably didn't pull the latest code, The code has been refactored with a PR which was merged earlier in the day, 
There are no files like these now:
src/Components/ViewOpportunity/Sections/FullTime/FullTime.js   

Please pull the latest code and do the changes to it. 

Overall, The changes that you have made are good.

**Comment by saumyasrivastava08** [comment]
@Manvityagi how to do that

**Comment by Manvityagi** [comment]
> @Manvityagi how to do that

# add upstream
$ git remote add upstream https://github.com/Girl-Code-It/Opportunity-Calendar-Frontend

# include all the latest changes from the remote repository
$ git fetch upstream
$ git merge upstream/develop

File: package.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
```
Comment (abdus) [inline package.json:7]: we already have a UI framework (`react-bootstrap`). I don't see a need for adding `@material-ui/core`

File: src/Components/ViewOpportunity/Sections/Dropdown/dropdown.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import Fade from '@material-ui/core/Fade';
```
Comment (abdus) [inline src/Components/ViewOpportunity/Sections/Dropdown/dropdown.js:5]: please check this out https://react-bootstrap.github.io/components/dropdowns/

File: src/Components/ViewOpportunity/Sections/Navbar.js [hunk 3]
```diff
@@ -1,56 +1,218 @@
...
+import InputBase from '@material-ui/core/InputBase';
+import Badge from '@material-ui/core/Badge';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import MenuIcon from '@material-ui/icons/Menu';
+import SearchIcon from '@material-ui/icons/Search';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import MailIcon from '@material-ui/icons/Mail';
+import NotificationsIcon from '@material-ui/icons/Notifications';
+import MoreIcon from '@material-ui/icons/MoreVert';
```
Comment (abdus) [inline src/Components/ViewOpportunity/Sections/Navbar.js:16]: Please replace these with [`react-bootstrap`](https://react-bootstrap.github.io/components/)

**Review by abdus — CHANGES_REQUESTED** [review]

**Comment by saumyasrivastava08** [comment]
@Manvityagi @abdus I will do the changes u told

File: src/Components/ViewOpportunity/Sections/Navbar.js (hunk 3 above)
Comment (saumyasrivastava08) [reply src/Components/ViewOpportunity/Sections/Navbar.js:16]: Sure

**Comment by saumyasrivastava08** [comment]
I have push the new changes but all the prechecks are failed

**Comment by abdus** [comment]
@saumyasrivastava08, which pull request should I consider? this one(#39) or pull #46 ? It's a bit confusing as the pull request description you have provided is not clear. 
Please let me know.

**Comment by saumyasrivastava08** [comment]
@abdus  consider the previous one mobile tab view

**Pull request closed without merging**